src/Symfony/Component/Lock/Store/MongoDbStore.php

```diff
@@ -91,7 +91,7 @@ class MongoDbStore implements PersistingStoreInterface
     public function __construct($mongo, array $options = [], float $initialTtl = 300.0)
     {
         $this->options = array_merge([
-            'gcProbablity' => 0.001,
+            'gcProbability' => 0.001,
```

Comment:
we cannot change the option name here as that would be a BC break for all users relying on the old name

Reply:
We need a deprecation notice when `gcProbablity` is defined, and its value copied into `gcProbability` when it is not already set.

Reply:
Just asking: what if we close this OR and leave the typo in the option name? The error is very minor, so would it be a big deal? Thanks.

Reply:
I think it indeed makes sense to rename this option as it will help people avoid WTF moments when they just type the option name without realising that we had a typo.

Reply:
I've committed some changes and rebased to 6.3. Please, tell me if I did it right. Thanks!

Reply:
LGTM, we need to update the `UPGRADE-6.3.md` file and the component's changelog though.

Reply:
I've updated the changelog and the upgrade guide. Thanks!